R/global.R

```diff
@@ -1,5 +1,16 @@
+
+# Activate virtual env
+Sys.unsetenv("RETICULATE_PYTHON")
+# Note, the name of the virtual environment is defined in the GH Actions workflow
+venv_name<-"virtual_env"
+reticulate::use_virtualenv(file.path(getwd(),venv_name))
+# We get a '126' error (non-executable) if we don't do this:
+system(sprintf("chmod -R +x %s", venv_name))
+
+
+
 if (interactive()) {
   options(shiny.port = 8100)
 } 
 
-OAUTH_LIST <- projectlive.modules::create_oauth_list()
+OAUTH_LIST <- projectlive.modules::create_oauth_list("oauth_config.yaml")
```

Comment:
@brucehoff clarifying question:
Currently "oauth_config.yaml" does not exist on the master branch. We have a template file on github and add in the credentials into the yaml file when pulled into the Shiny server. Historically I have done this to keep the oauth information private. 
1) Would the missing file throw an error when the PR is merged in and the app starts rerouting to the new url?
2) Would you recommend adding in the yaml file and credentials and store that on the github main branch (currently it is on lastpass) ?

Reply:
just noticed now that this file is probably being created automatically in line 67 in .github/workflows/shinyapps_deploy.yml .. can you please confirm if my understanding is correct?

Reply:
Yes, exactly.  No need to add the file in the master branch.